depends/toolchain.cmake.in

```diff
@@ -138,27 +140,31 @@ if(NOT WITH_SQLITE AND "@no_sqlite@" STREQUAL "1")
   set(WITH_SQLITE OFF CACHE STRING "Enable SQLite wallet support.")
 endif()
 
-if(NOT WITH_MINIUPNPC AND "@no_upnp@" STREQUAL "1")
-  set(WITH_MINIUPNPC OFF CACHE STRING "Enable UPnP.")
+if("@no_upnp@")
+  set(WITH_MINIUPNPC OFF CACHE BOOL "")
```

Comment:
I am not sure how those `"@no_foo@"` are generated, but there is some inconsistency between upnp and the others. For example, `CMakeLists.txt` defines `WITH_NATPMP` and in `depends/toolchain.cmake.in` the check is using `no_natpmp`. But for upnp: `CMakeLists.txt`: `WITH_MINIUPNPC`, `depends/toolchain.cmake.in`: `no_upnp`.

Reply:
> I am not sure how those "@no_foo@" are generated...

https://github.com/hebasto/bitcoin/blob/d66f911b6863bb3b20e8e28eb14edceb9db0d555/depends/Makefile#L288-L298

> ... but there is some inconsistency between upnp and the others. For example, `CMakeLists.txt` defines `WITH_NATPMP` and in `depends/toolchain.cmake.in` the check is using `no_natpmp`. But for upnp: `CMakeLists.txt`: `WITH_MINIUPNPC`, `depends/toolchain.cmake.in`: `no_upnp`.

You mean, `WITH_MINIUPNPC` should be renamed to `WITH_UPNP`?

Reply:
> You mean, `WITH_MINIUPNPC` should be renamed to `WITH_UPNP`?

Either this or change the above to `-e 's|@no_miniupnpc@|$(NO_MINIUPNPC)|'`. No strong opinion, but it is nice to be more consistent.